lib/CL/devices/common.c

```diff
@@ -1864,6 +1864,8 @@ static const cl_name_version OPENCL_FEATURES[] = {
   { CL_MAKE_VERSION (3, 0, 0), "__opencl_c_int64" },
   { CL_MAKE_VERSION (3, 0, 0), "__opencl_c_program_scope_global_variables" },
   { CL_MAKE_VERSION (3, 0, 0), "__opencl_c_generic_address_space" },
+  { CL_MAKE_VERSION (3, 0, 0), "__opencl_c_subgroups" },
+  { CL_MAKE_VERSION (3, 0, 0), "__opencl_c_work_group_collective_functions" },
```

Comment:
The Level0 driver supports them, since the API accepts SPIR-V which uses them. Although the documentation implies that it needs an extension (`ZE_extension_subgroups`) so i might have to add a check for that. This static array is only the list of  "recognized" features, the actual features supported by any device are stored in `device->features` string, so `__opencl_c_work_group_collective_functions` shouldn't be reported for CPU device.